packages/paste-core/components/code-block/package.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "@twilio-paste/code-block",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "production",
+  "description": "A component used to display readable blocks of code.",
```

```suggestion
  "description": "A Code Block is a component used to display readable blocks of code.",
```
suggestion: matches our pattern for component descriptions